.changeset/rotten-birds-build.md

```diff
@@ -0,0 +1,6 @@
+---
+'@backstage/plugin-microsoft-calendar': patch
+---
+
+created new plugin @backstage/plugin-microsoft-calendar and added it to plugins directory
```

Comment:
It would be really great to include how to use it here as this changeset will get used as the release notes. For those upgrading theres a good chance this will be the way they learn about this plugin first.